x/gov/spec/02_state.md

```diff
@@ -4,6 +4,48 @@ order: 2
 
 # State
 
+## Proposals
+
+`Proposal` objects are used to tally votes and generally track the proposal's state.
+They contain an array of arbitrary `sdk.Msg`'s which the governance module will attempt
+to resolve and then execute if the proposal passes. `Proposal`'s are identified by a
+unique id and contains a series of timestamps: `submit_time`, `deposit_end_time`,
+`voting_start_time`, `voting_end_time` which track the lifecycle of a proposal
+
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0/proto/cosmos/gov/v1beta2/gov.proto#L55-L77
```

Comment:
This is a 404. We might need to use a github commit hash instead of 0.40.0

Reply:
True. Thanks for pointing that out